src/main/java/nl/b3p/tailormap/api/persistence/GeoService.java

```diff
@@ -416,21 +440,21 @@ public TMFeatureType findFeatureTypeForLayer(
   /**
    * Remove all parameters from the URL that are listed in {@link #REMOVE_PARAMS}.
    *
-   * @param url URL to sanitise
-   * @return sanitised URL
+   * @param url URL to sanitize
```

Comment:
```suggestion
   * @param url URL to sanitise
```

https://grammarist.com/spelling/sanitise-sanitize/